app/inputs/dropdown_select_input.rb

```diff
@@ -0,0 +1,41 @@
+class DropdownSelectInput < SimpleForm::Inputs::CollectionInput
+  include ActionView::Helpers::TagHelper
+  include ActionView::Context
+
+  def current_value
+    input_html_options[:value] || object.try(:send, attribute_name)
+  end
+
+  def selected_option
+    if current_value
+      collection.find { |x| x[1] == current_value }
+    else
+      collection.first
+    end
+  end
+
+  def input(wrapper_options = nil)
```

Comment:
Unused method argument - `wrapper_options`. If it's necessary, use `_` or `_wrapper_options` as an argument name to indicate that it won't be used. You can also write as `input(*)` if you want the method to accept any arguments but don't care about them.